ArduPlane/RC_Channel.cpp

```diff
@@ -34,6 +34,7 @@ void RC_Channel_Plane::init_aux_function(const RC_Channel::aux_func_t ch_option,
     // the following functions do not need to be initialised:
     case ARMDISARM:
     case INVERTED:
+    case LANDING_GEAR:
```

Comment:
This should be in RC_Channel/RC_Channel.cpp (and the one in Copter removed)

Reply:
done